test/functionalTest/cases/3727_avoid_triggering_notification_if_metadata_changes_but_not_the_attribute_value_itself/notifyOnMetadataChange_basic_crud.test

```diff
@@ -0,0 +1,435 @@
+        "type": "Thing"
+      }
+    ],
+    "condition": {
+      "attrs": [
+        "A1"
+      ]
+    }
+  }
+}'
+orionCurl --url /v2/subscriptions/$subId -X PATCH --payload "$payload"
+echo
+echo
+
+
+echo "10. GET subscription and see notifyOnMetadataChange true"
+echo "========================================================"
+orionCurl --url /v2/subscriptions/$subId
+echo
+echo
+
+
+
+
+
+
+--REGEXPECT--
+01. POST /v2/subscriptions, to without notifyOnMetadataChange (defaults to true)
+================================================================================
+HTTP/1.1 201 Created
+Content-Length: 0
+Location: /v2/subscriptions/REGEX([0-9a-f]{24})
+Fiware-Correlator: REGEX([0-9a-f\-]{36})
+Date: REGEX(.*)
+
+
+
+02. GET subscription and see notifyOnMetadataChange true
+========================================================
+HTTP/1.1 200 OK
+Content-Length: 304
+Content-Type: application/json
+Fiware-Correlator: REGEX([0-9a-f\-]{36})
+Date: REGEX(.*)
+
+{
+    "id": "REGEX([0-9a-f]{24})",
+    "notification": {
+        "attrs": [],
+        "attrsFormat": "normalized",
+        "covered": false,
+        "http": {
+            "url": "http://localhost:9997/notify"
+        },
+        "onlyChangedAttrs": false
+    },
+    "status": "active",
+    "subject": {
+        "condition": {
+            "attrs": [
+                "A1"
+            ],
+            "notifyOnMetadataChange": true
+        },
+        "entities": [
+            {
+                "id": "E1",
+                "type": "Thing"
+            }
+        ]
+    }
+}
+
+
+03. PATCH /v2/subscriptions, with notifyOnMetadataChange: false
+===============================================================
+HTTP/1.1 204 No Content
+Fiware-Correlator: REGEX([0-9a-f\-]{36})
+Date: REGEX(.*)
+
+
+
+04. GET subscription and see notifyOnMetadataChange false
+=========================================================
+HTTP/1.1 200 OK
+Content-Length: 305
+Content-Type: application/json
+Fiware-Correlator: REGEX([0-9a-f\-]{36})
+Date: REGEX(.*)
+
+{
+    "id": "REGEX([0-9a-f]{24})",
+    "notification": {
+        "attrs": [],
+        "attrsFormat": "normalized",
+        "covered": false,
+        "http": {
+            "url": "http://localhost:9997/notify"
+        },
+        "onlyChangedAttrs": false
+    },
+    "status": "active",
+    "subject": {
+        "condition": {
+            "attrs": [
+                "A1"
+            ],
+            "notifyOnMetadataChange": false
+        },
+        "entities": [
+            {
+                "id": "E1",
+                "type": "Thing"
+            }
+        ]
+    }
+}
+
+
+05. PATCH /v2/subscriptions, with notifyOnMetadataChange: true
+==============================================================
+HTTP/1.1 204 No Content
+Fiware-Correlator: REGEX([0-9a-f\-]{36})
+Date: REGEX(.*)
+
+
+
+06. GET subscription and see notifyOnMetadataChange true
+==============================================================
```

Comment:
```suggestion
06. GET subscription and see notifyOnMetadataChange true
========================================================
```

Reply:
Fixed in [3ba1ead](https://github.com/telefonicaid/fiware-orion/pull/4310/commits/3ba1ead3128937e80932ee1f3210bacc23f3d475)